bfdd/bfd_packet.c

```diff
@@ -768,6 +768,39 @@ static void cp_debug(bool mhop, struct sockaddr_any *peer,
 		   mhop ? "yes" : "no", peerstr, localstr, portstr, vrfstr);
 }
 
+static int bfd_check_auth(struct bfd_session *bfd, struct bfd_pkt *cp){
+
+	if(CHECK_FLAG(cp->flags, BFD_ABIT)) {
```

Comment:
If `auth` is set then it should always return `false` because it is not implemented.